apps/aeutils/priv/aeternity_config_schema.json

```diff
@@ -47,6 +47,10 @@
                         }
                     },
                     "default": []
+                },
+                "custom_prefunded_accs_file" : {
```

Comment:
why plugins suddenly started polluting the node configuration ?

Reply:
> why plugins suddenly started polluting the node configuration ?

The way it normally works is that the prefunded accounts are created through hardcoded files.
This way, it's possible to create a custom chain using unmodified source without inheriting a truckload of predefined accounts specific to mainnet.

Reply:
Nevermind, that seems just ENV->config variable change, unrelated to the plugin update